t/syntax/00-lib.t

```diff
@@ -26,7 +26,8 @@ fi
 tests 3
 
 run_pass "${TEST_KEY_BASE}" \
-    pep8 "${ROSE_HOME}/lib/python/rose" "${ROSE_HOME}/lib/python/rosie"
+    pep8 --ignore=E402,E731 \
```

Comment:
I'm not sure we need the gtk.require part - I think it may be legacy from earlier GTK.